app/src/main/res/values-ru/strings.xml

```diff
@@ -284,11 +283,8 @@
     <string name="no_commits">Фиксации отсутствуют</string>
     <string name="no_contributors">Вкладчики отсутствуют</string>
     <string name="no_releases">Сборки отсутствуют</string>
-    <string name="no_closed_issues">Закрытые Тикеты отсутствуют</string>
-    <string name="no_opened_issues">Открытые Тикеты отсутствуют</string>
     <string name="no_events">События отсутствуют</string>
-    <string name="no_open_pull_requests">Запросы на слияние отсутствуют</string>
-    <string name="no_closed_pull_request">Закрытые Запросы на слияние отсутствуют</string>
+    <string name="no_pull_requests">Запросы на слияние отсутствуют</string>
```

Comment:
here too.

Reply:
Pull Requests can be translated into languages without a loss of the meaning. It's a correct variant.